Core/clim-basic/extended-streams/stream-input.lisp

```diff
@@ -440,7 +435,7 @@ keys read."))
 
 (defmethod stream-read-gesture ((stream #.*string-input-stream-class*)
                                 &key peek-p
-                                &allow-other-keys)
+                                  &allow-other-keys)
```

Comment:
Do not indent or include in previous line.

Reply:
wrapped to previous line.